webplugin/js/app/kommunicate-ui.js

```diff
@@ -1803,4 +1803,220 @@ KommunicateUI = {
             }
         );
     },
+    initRecorder: function(){
+        const LIVE_OUTPUT = false; // a feature to live output the recording voice to the speaker
```

Comment:
verify if const is working on all major browsers

Reply:
![Screenshot 2022-01-05 at 4 56 54 PM](https://user-images.githubusercontent.com/19753380/148210340-0c02f9d5-e47b-44f5-bc44-563454463547.png)

For IE and opera mini their is partial support.